scalafmt-tests/src/test/resources/scala3/OptionalBraces_keep.stat

```diff
@@ -1414,7 +1414,8 @@ val f: Int => Int = (x: Int) => {
   x * x
 }
 >>>
-val f: Int => Int = (x: Int) => x * x
+val f: Int => Int = (x: Int) =>
+  x * x
```

Comment:
do we need to add tests for which isCurlyLambda would be true?

Reply:
there is a bunch of tests which already test it (they broke when I removed it)